src/arrays/ArrayUtil.java

```diff
@@ -46,15 +85,28 @@ public static boolean contains(int[] array, int value) {
      * @return the first index of value in the array, if array does not contain given element return -1.
      */
     public static int indexOf(int[] array, int value) {
+        for(int i=0; i< array.length; i++){
+            if (array[i]==value){
+                return i;
+            }
+
+        }
 
-        return 0;
+        return -1;
     }
 
+
     /**
      * Remove the element by the given index.
      * Print new array
      */
     public static void remove(int[] array, int index) {
+        int[] newArray=new int[array.length-index];
+        for(int i=index;i< array.length-index; i++){
+            newArray[i]=array[i-index];{
+                System.out.println(newArray[i]);
+            }
+        }
 
     }
```

Comment:
You wrote wrong.
You can write like this, for example`
 int result[] = new int[array.length - 1];
        for (int i = 0, j = 0; i < array.length; i++) {
            if (i != index) {
                result[j++] = array[i];
            }
        }